### ISSUE TITLE
Could not execute the first step on a computer with username written in non-ASCII characters

### ISSUE THREAD

**Issue by MariiaBash** [SoloveyMaria/community#2]
I followed the instructions and ran install_windows.bat multiple times. Each time, it downloaded Conda and exited without any errors. However, I couldn't proceed past this step. The script consistently downloaded Conda and then exited. Afterwards, I attempted to manually download and install Conda, but encountered the following error. It would be helpful if you could include a warning stating that Conda is not compatible when the installation path contains non-ASCII characters.

![image](https://github.com/SoloveyMaria/community/assets/135337166/75983d75-897f-4f51-add7-e1fee7e91e4a)

**Comment by MariiaBash** [comment]
I changed the username  so that installation path only consists of ASCII characters and ran install_windows.bat again. It tried to create and solve an environment but then exited. I ran it again and following error occured.
(in the last lines it states 
'Rscript' is not recognized as an internal or external command, operable program or batch file.
'jupyter' is not recognized as an internal or external command, operable program or batch file.
)
![6](https://github.com/SoloveyMaria/community/assets/135337166/8212a9c4-1ed9-427b-b1ef-5b6ebfad830c)

I then ran "conda info --envs" and this is what I got. The community_tutorial evironment was indeed not created.
![7](https://github.com/SoloveyMaria/community/assets/135337166/2390dfb2-2c99-46e2-b3d7-72816f1ea33f)
